src/common/commonutils/SshUtils.c

```diff
@@ -205,13 +205,11 @@ static int IsSshServerActive(void* log)
 // See https://www.openssh.com/txt/release-8.2, quote: "add an Include sshd_config keyword that allows including additional configuration files"
 static int IsSshConfigIncludeSupported(void* log)
```

Comment:
>IsSshConfigIncludeSupported [](http://example.com/codeflow?start=11&length=27)

Please see the comment on lines 226-228. 

This function was tested on all target distros, both supporting -V and not and it was found to work well. We also do not have any customer reports telling us they hit a problem with this, which is specific to the 'SSH security posture for Linux' policy which is now shipped in GA, Unless there is a clear, reproducible bug, of high priority, sorry, but I won't agree with this change.

Please elaborate on the bug repro steps so that we could repro in house and investigate. 

Until then, the other fixes look good, but won't signoff until we agree on this change here for IsSshConfigIncludeSupported.

Reply:
If you could split this in 2 PRs, the other one will be accepted right away and then we can keep this other one for further discussion.

Reply:
I'm dropping this commit for now and will revisit again next week.